resources/public/dashboard/index.html

```diff
@@ -257,19 +257,14 @@
        url: "/metrics/v1/mbeans/puppetlabs.puppetdb.storage:name=gc-time",
        format: d3.format(",.3s"),
        snag: getPercentileSnag},
-      {description: "DLO Compression",
-       addendum: "round trip time, seconds",
-       url: "/metrics/v1/mbeans/puppetlabs.puppetdb.dlo:name=global.compression",
-       format: d3.format(",.3s"),
-       snag: getPercentileSnag},
       {description: "DLO Size on Disk",
        addendum: "bytes",
-       url: "/metrics/v1/mbeans/puppetlabs.puppetdb.dlo:name=global.filesize",
```

Comment:
This just changes it to match the new dlo names; there we now only have "all" or a command name, each with a total size and count, but happy to change that if we prefer something else.